requirements.txt

```diff
@@ -1,10 +1,9 @@
 #
 # Requirements for wger for production
 #
-
-Django>=1.9,<1.10
-django-recaptcha
-reportlab
+Django==1.9   # changed this line
+django-recaptcha==1.3.1  # changed this line
```

Comment:
You can remove these comments in the file for now according to another task, we are going to have explicitly defined requirements versions to install.